samples/DataGridListSample.js

```diff
@@ -87,8 +87,8 @@ enyo.kind({
 enyo.kind({
 	name: "moon.sample.GridSampleItem",
 	kind: "moon.GridListImageItem",
-	mixins: ["moon.SelectionOverlaySupport"],
-	selectionOverlayVerticalOffset: 35,
```

Comment:
I agree. It would be nice to see the examples two approaches, instead of just one.